util/misc.py

```diff
@@ -353,3 +354,109 @@ def get_intervals(self, c=None):
                 right = right - 1
                 features = list(self.get_features(c, left, right))
                 yield (c, left, right, len(features), features)
+
+
+def available_cpu_count():
```

Comment:
It's a heuristic. psutil is not *required*; it's one option attempted. I can simplify it to use the Snakemake version; it would be less to maintain.